crates/model/src/order.rs

```diff
@@ -61,7 +61,7 @@ pub enum OrderStatus {
 
 impl Order {
     pub fn from_order_creation(
-        order_creation: OrderCreation,
+        order_creation: &OrderCreation,
```

Comment:
What is the reason for this change?
They way it was before makes more sense to me.